tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp

```diff
@@ -786,6 +786,7 @@ void Sprite3DEffectTest::addNewSpriteWithCoords(Vec2 p)
     material->setTechnique("outline_noneskinned");
     sprite->setMaterial(material);
     sprite->setScale(6.f);
+    Director::getInstance()->getTextureCache()->removeUnusedTextures();
```

Comment:
This is an issue while using Sprite3D. IMO, moving it to Texture2DTest is probably not a good idea. 
As I know, one of the purpose of test case is for easily find out issues before releasing a new version. And our QA team will check display of every test case. When he/she checks `Sprite3DEffectTest` and found the texture is missing, he/she will report an issue to us.
Therefore,
* If a new test case is needed indeed, would it be better to add it in Sprite3DTest or BugTests?
* New test case will contain duplicated code, probably all the same as `Sprite3DEffectTest` except adding a new line of `removeUnusedTextures`.

@ricardoquesada , anyway, I will add it if you wish to. :)